src/main/java/school/lemon/changerequest/java/introduction/hw2/Task8.java

```diff
@@ -3,6 +3,21 @@
 public class Task8 {
 
     public static int[] evenOdd(int[] arr) {
-        return null;
+
+        int sortArr[] = new int[arr.length];
+        int j = 0;
+        for (int anArr : arr) {
+            if (anArr % 2 == 0)
+                sortArr[j++] = arr[anArr];
+        }
+
+        int n = 0;
+        for (int newArr: arr) {
+            if (newArr % 2 != 0) {
```

Comment:
The only possible values of any number % 2 are 0 and 1, so it'll be clearer if you'll check only those values.